code/part-one/tests/01-Signing.js

```diff
@@ -31,6 +31,33 @@ describe('Signing module', function() {
     });
   });
 
+  describe('getPublicKey', function() {
+    let privateKey = null;
+    let publicKey = null
+
+    beforeEach(function() {
+      privateKey = signing.createPrivateKey();
+      publicKey = signing.getPublicKey(privateKey);
+    });
+
+    it('should return a hex string', function() {
+      expect(publicKey).to.be.a('string').and.match(/^[0-9a-f]*$/);
```

Comment:
Should we try `expect(publicKey).to.be.a.hexString;` here as well (if it works)?